meta.yaml

```diff
@@ -17,8 +17,12 @@ spec:
     env:
       - name: CHE_API
         value: $(CHE_API_INTERNAL)
```

Comment:
Do we need CHE_API env var at all?

Reply:
I'll double check

Reply:
I don't think we really use it.  However, this requires a `che-workspace-telemetry-client` change to remove it from the `Configuration` class.

Reply:
ok, let's probably do it in a separate PR after deprecating https://github.com/redhat-developer/che-workspace-telemetry-woopra-backend